src/commcare_cloud/ansible/roles/commcarehq/templates/celery_bash_runner.sh.j2

```diff
@@ -44,7 +44,8 @@ HOSTNAME_ARG+=".${TIMESTAMP}_timestamp"
 HOSTNAME_PARTS=(${HOSTNAME_ARG//=/ })
 HOSTNAME=${HOSTNAME_PARTS[1]}
 
-{{ virtualenv_home }}/bin/python {{ item.get('python_options', '') }} {{ code_home }}/manage.py celery worker ${HOSTNAME_ARG} ${ARGS}  &
+source {{ virtualenv_home }}/bin/activate
```

Comment:
Why is this being sourced? I'm hesitant to make this change if it's not necessary, because I believe it would affect the entire script/shell. Before when it's not sourced, it should only affect the command (same for supervisor changes)

Reply:
Thanks, updated